tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/featured_services_block.html

```diff
@@ -1,22 +1,26 @@
 {% load wagtailcore_tags wagtailimages_tags %}
-<div>
+<div class="grid__featured-services {% if value.intro %}featured-services--with-intro{% endif %}">
```

Comment:
as an option, we could simplify it by adding both the title and the intro into a div container with a margin-bottom of 60px

Reply:
definitely : )